### DISCUSSION TITLE
Support multiple actions

### DISCUSSION THREAD

**Discussion by byalextran** [rxhanson/Multitouch-Community#162] [Ideas]
I've been using BetterTouchTool and like that Multitouch is a BTT-light app. BTT seems a bit overkill for the things I use it for. 

However, the one thing missing from Multitouch that's preventing me from making the switch is the ability to chain multiple actions together. 

A prime example of this for me is the ability to have a shortcut do two things:

1. Move window to next display. 
2. Full screen the window (necessary since I have multiple monitors of different resolutions). 

Right now I can do one or the other with Multitouch, but not both at once (unless I'm missing something). 

So would love to see support for multiple actions.

**Comment by rxhanson** [comment]
Thanks for trying Multitouch & for the feedback. Indeed, keeping Multitouch simple is one of the main design decisions, and this is one of the reasons why I have left out chaining actions. I would prefer to add common multi-step actions as opposed to adding in a UI for chaining actions, even though the lists of actions can feel a bit unwieldy. I have plans to make the lists of actions a bit more readable with headers & grouping within the list.

With that said, how about a couple of window management items for next display maximized and prev display maximized? Are there other chained actions that you use?

**Comment by byalextran** [comment]
For right now that's the only chained sequence of actions I can think of. If you were able to add next/previous displayed maximized, that would be enough to get me to switch over!

**Reply by rxhanson** [reply]
Here's a quick beta with next & prev, maximize:
https://www.multitouch.app/beta/multitouch1.27.15.dmg

**Reply by byalextran** [reply]
clutch! seems to be working well. i've disabled BTT for now and will use this for a bit and see if i run into any bugs and/or discover any other missing multi-step actions. 

appreciate the quick response/beta!
